website/docs/docs/get-started/develop-in-the-cloud.md

```diff
@@ -85,33 +85,23 @@ New to dbt? Check out our [Getting Started guide](/docs/get-started/getting-star
 
 In order to start experiencing the great features of the Cloud IDE, you need to first set up a [dbt Cloud development environment](/docs/collaborate/environments/dbt-cloud-environments) and [developer credentials](#developer-credentials). If you're creating a new project, you will automatically configure this during the project setup. 
 
-Review the following steps to set up your developer credentials:
-
-### Developer credentials
-
 The IDE uses developer credentials to connect to your data platform. These developer credentials should be specific to your user and they should *not* be super user credentials or the same credentials that you use for your production deployment of dbt.
 
-Follow the below steps to set up your developer credentials:
+Review the following steps to set up your developer credentials:
 
 1. Go to your [**Credentials**](https://cloud.getdbt.com/next/settings/profile#credentials) under the **Your Profile** settings.
 2. Select the relevant project in the list.
 3. Click **Edit** on the bottom right of the page.
 4. Fill in the details in **Development Credentials**, including Dataset and Target Name. 
 5. Click **Save.**
 
-Great job, you should now be able to access the Cloud IDE by clicking **Develop** on the navigation to start developing!
-
 <Lightbox src="/img/docs/dbt-cloud/refresh-ide/dev-credentials.png" width="100" height="100" title="Configure developer credentials in your Profile"/>
 
-
-### Access the Cloud IDE
-
 Now that you've set up your development environment and credentials, you should be able to access the Cloud IDE:
```

Comment:
Is there a reason these steps are separate from setting up dev credentials? I'm not suggesting we combine them, just curious why they are separate. If we keep them separate, I made a suggestion in the into paragraph about making it clear what you will learn here. 

We could simplify this as well.
```suggestion
Access the Cloud IDE:
```

Reply:
good question and my initial thought was because they different steps on diff pages, i wanted to make it clear to the user that they have to do xx things on this page and xx things on that page. however, your question makes me believe that if someone has to always create their dev credentials and then go to **develop** to access the ide, maybe i should combine the steps. I've done that any to see how it flows!